tests/layer_tests/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+project(layer_tests)
+
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/ DESTINATION layer_tests_openvino COMPONENT tests EXCLUDE_FROM_ALL)
```

Comment:
please replace DESTINATION folder `layer_tests_openvino` to `tests\layer_tests_openvino`